### PR TITLE
Site styling refresh

### PR DESCRIPTION
**Pull request by chriswilk**
I've implemented Joshua's style changes and will be promoting this in a blog post soon.

## Desktop - homepage:
screencapture-localhost-18080-1479893127941

## Mobile - homepage
screencapture-localhost-18080-1479893185619

## Desktop - access page
screencapture-localhost-18080-access-1479893164846

## Mobile - access page
screencapture-localhost-18080-access-1479893238178

### CODE REVIEW

**Comment by oilnam** [comment]
Nice! 👏 

I liked the King's Place picture but that one is also cool, although maybe slightly less significant?

Also now that we have 2.1M pieces of content we can replace the _access over 1.9M pieces of content_ line everywhere with _access over 2M pieces of content_ which is a nicely rounded number.

**Comment by chriswilk** [comment]
@oilnam Cool

King's place definitely doesn't make sense. I've got the designers thinking about the hero image. Northern lights one was signed off by Alex Breuer already.

I took the number of articles that are available on the developer tier.

File: access/index.html [hunk 1]
```diff
@@ -16,108 +16,118 @@
...
+   <div class="container" >
+   <div class="row">
+      <div class="col-sm-11">
+           <ul class="nav navbar ">
+              <li><a href="https://groups.google.com/forum/#!forum/guardian-api-talk">Support</a></li>
+              <li><a href="https://www.theguardian.com/info/developer-blog">Developer Blog</a></li>
+              <li><a href="https://developers.theguardian.com/join-the-team.html">Join Us</a></li>
+              <li><a href="https://twitter.com/openplatform">Follow Us</a></li>
+          </ul>
+          <p>© 2015 Guardian News and Media Limited or its affiliated companies. All rights reserved.</p>
```
Comment (mchv) [inline access/index.html]: `2016` :)

**Comment by chriswilk** [comment]
@mchv good spot Mariot! fixed

**Comment by mchv** [comment]
👍 This seems good enough to be merged!
if we detect problems after we can fix them with PR

**Comment by mchv** [comment]
@chriswilk this is live, little things we could do to improve:
 - update [bonobo register page](https://bonobo.capi.gutools.co.uk/register/developer) to match new style
- update the [explorer](http://explorer.capi.gutools.co.uk/) json colors to match the new style

- fix the missing phone image `http://open-platform.theguardian.com/public/img/Phones.svg`
screen shot 2016-11-24 at 09 35 28

- fix the svg, some white lines seems to not display
screen shot 2016-11-24 at 09 38 21
screen shot 2016-11-24 at 09 38 39

**Pull request merged**